addons/popochiu/editor/main_dock/tab_audio/tab_audio.gd

```diff
@@ -0,0 +1,353 @@
+	ar.stream_player = _asp
+	
+	ar.deleted.connect(_audio_cue_deleted)
+	ar.clicked.connect(_on_row_clicked)
+	
+	return ar
+
+
+func _create_audio_file_row(file_path: String) -> void:
+	var ar: HBoxContainer = POPOCHIU_AUDIO_ROW_SCENE.instantiate()
+	
+	ar.name = file_path.get_file().get_basename()
+	ar.path = file_path
+	ar.file_name = file_path.get_file()
+	ar.audio_tab = self
+	ar.stream_player = _asp
+	
+	ar.group_selected.connect(_create_audio_cue.bind(file_path, ar))
+	
+	unassigned_group.add(ar, true)
+	_audio_files_to_assign[file_path] = ar
+
+
+func _read_directory(dir: EditorFileSystemDirectory) -> void:
+	if not dir: return
+	
+	if dir.get_subdir_count():
+		for d in dir.get_subdir_count():
+			# Look out sub directories
+			_read_directory(dir.get_subdir(d))
+
+		# Look out the directory files
+		_read_files(dir)
+	else:
+		_read_files(dir)
+
+
+func _read_files(dir: EditorFileSystemDirectory) -> void:
+	for idx in dir.get_file_count():
+		var file_name = dir.get_file(idx)
+		
+		if not file_name.get_extension() in ["ogg", "mp3", "wav", "opus"]:
+			continue
+		
+		if dir.get_file_path(idx) in _audio_files_in_group\
+		or dir.get_file_path(idx) in _audio_files_to_assign:
+			# Don't put in the list an audio file already assigned to a PopochiuAudioCue in
+			# popochiu_data.cfg
+			continue
+			
+		# Check if the file prefix matches any of the prefixes defined to automatically assign it to
+		# a group
+		var target := PopochiuResources.AudioTypes.NONE
+		var prefix := file_name.get_slice(PopochiuConfig.get_prefix_character(), 0)
+		
+		if prefix in PopochiuConfig.get_music_prefixes():
+			target = PopochiuResources.AudioTypes.MUSIC
+		elif prefix in PopochiuConfig.get_sound_effect_prefixes():
+			target = PopochiuResources.AudioTypes.SOUND_EFFECT
+		elif prefix in PopochiuConfig.get_voice_prefixes():
+			target = PopochiuResources.AudioTypes.VOICE
+		elif prefix in PopochiuConfig.get_ui_prefixes():
+			target = PopochiuResources.AudioTypes.UI
+		
+		if target != PopochiuResources.AudioTypes.NONE:
+			_audio_cues_to_create.append([target, dir.get_file_path(idx)])
+		else:
+			# Put the file in the Unassigned group so devs can manually choose the group to which
+			# the audio file should be assigned
+			_create_audio_file_row(dir.get_file_path(idx))
+
+
+func _create_audio_cue(type: int, path: String, audio_row: Container = null) -> void:
+	var cue_name := path.get_file().get_basename()
+	var cue_file_name := cue_name
+	cue_file_name += ".tres"
+	
+	# Create the AudioCue and save it in the file system
+	var ac: AudioCue
+	
+	match type:
+		PopochiuResources.AudioTypes.MUSIC:
+			ac = AudioCueMusic.new()
+			ac.loop = true
+		_:
+			ac = AudioCueSound.new()
+	
+	var stream: AudioStream = load(path)
+	ac.audio = stream
+	ac.resource_name = cue_name.to_lower()
+	
+	var error: int = ResourceSaver.save(ac, "%s/%s" % [path.get_base_dir(), cue_file_name])
+	
+	assert(error == OK) #,"[Popochiu] Can't save AudioCue: %s" % cue_file_name)
```

Comment:
Did you miss printing a message here?

Reply:
Hahahaha. Looks like. I'll make the update.